src/index.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import './style.css';
+import { addNewTask, deleteTask, renderTasks } from './modules/new.js';
+import { editTaskDescription, saveEditedTaskDescription } from './modules/edit.js';
+/* eslint-enable */
+
```

Comment:
- Hi, Can you fix the linter manually? Because it is prohibited to disable [Linter](https://github.com/microverseinc/linters-config). A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs